dolphinscheduler-alert-plugin/dolphinscheduler-alert-http/src/main/java/org/apache/dolphinscheduler/plugin/alert/http/HttpSender.java

```diff
@@ -110,17 +114,17 @@ public AlertResult send(String msg) {
     }
 
     private void createHttpRequest(String msg) {
-
         if (REQUEST_TYPE_POST.equals(requestType)) {
             httpRequest = new HttpPost(url);
+            setHeader();
             //POST request add param in request body
             setMsgInRequestBody(msg);
```

Comment:
I think it can be changed as follows：
我觉得可以改为如下：
String bodyParams = setMsgInRequestBody(msg);
StringEntity entity = new StringEntity(bodyParams, DEFAULT_CHARSET);
 ((HttpPost)httpRequest).setEntity(entity);

Reply:
Header is required set by user on ui page,I just get header by user set.
header是用户在界面上必须要配置的，我只是从用户配置的header取的值
![微信图片_20210301140515.png](http://ww1.sinaimg.cn/large/005Cc2n2ly1go4de8hh1wj30f50gfgm5.jpg)

Reply:
> Header is required set by user on ui page,I just get header by user set.
> header是用户在界面上必须要配置的，我只是从用户配置的header取的值
> ![微信图片_20210301140515.png](https://camo.githubusercontent.com/b92fc909fd6451e7e289b0c73c85fb43b5145a529ba9983bde777db98953ad7f/687474703a2f2f7777312e73696e61696d672e636e2f6c617267652f3030354363326e326c7931676f34646538686831776a33306635306766676d352e6a7067)

The Content-Type in the header is not mandatory, so it is best not to get the value from the header. The body parameter must be in JSON format
header中Content-Type 不是必填项，所以最好不要从header中获取这个值，body 参数必须是json格式的.

Reply:
ok,I'll change that